lib/plugins/awsDeploy/awsDeploy.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const validateInput = require('./lib/validateInput');
+const createStack = require('./lib/createStack');
+const deployFunctions = require('./lib/deployFunctions');
+const compileFunctions = require('./lib/compileFunctions');
+const updateStack = require('./lib/updateStack');
+
+const AWS = require('aws-sdk');
+
+class awsDeploy {
+  constructor(serverless) {
+    this.serverless = serverless;
+
+    Object.assign(this, validateInput,createStack, deployFunctions, compileFunctions, updateStack);
```

Comment:
missing comma after "validateInput," Please take a look at the latest build and fix all eslint issues in files that are touched here: https://travis-ci.org/serverless/serverless/builds/134452095